content/docs/add-react-to-a-website.md

```diff
@@ -1,202 +1,202 @@
+- [Ավելացրու React-ը մեկ րոպեում](#add-react-in-one-minute)
+- [Ոչ պարտադիր. փորձիր React-ը JSX-ով](#optional-try-react-with-jsx) (առանց bundler-ի անհրաժեշտության)
 
-## Add React in One Minute {#add-react-in-one-minute}
+## Ավելացրու React-ը մեկ րոպեում {#add-react-in-one-minute}
 
-In this section, we will show how to add a React component to an existing HTML page. You can follow along with your own website, or create an empty HTML file to practice.
+Այս բաժնում, մենք ցույց կտանք, թե ինչպես ավելացնել React կոմպոնենտ գոյություն ունեցող HTML էջ։ Վարժանքի համար կարող եք հետևել քայլերին\` կիրառելով նրանք ձեր սեփական վեբկայքի վրա կամ ստեղծել դատարտ HTML ֆայլ։
 
-There will be no complicated tools or install requirements -- **to complete this section, you only need an internet connection, and a minute of your time.**
+Բարդ գործիքներ կամ տեղադրում չկա. **այս բաժինը վերջացնելու համար, ձեզ միայն անհրաժեշտ հասանելիություն համացանցին և հատկացնել մեկ րոպե ձեր ժամանակից։**
 
-Optional: [Download the full example (2KB zipped)](https://gist.github.com/gaearon/6668a1f6986742109c00a581ce704605/archive/f6c882b6ae18bde42dcf6fdb751aae93495a2275.zip)
+Ոչ պարտադիր. [Ներբեռնեք ամբողջ օրինակը (արխիվացրած վիճակով 2ԿԲ)](https://gist.github.com/gaearon/6668a1f6986742109c00a581ce704605/archive/f6c882b6ae18bde42dcf6fdb751aae93495a2275.zip)
 
-### Step 1: Add a DOM Container to the HTML {#step-1-add-a-dom-container-to-the-html}
+### Քայլ 1. Ավելացրու DOM կոնտեյներ HTML-ում {#step-1-add-a-dom-container-to-the-html}
 
-First, open the HTML page you want to edit. Add an empty `<div>` tag to mark the spot where you want to display something with React. For example:
+Սկզբից բացիր HTML ֆայլը, որը ցանկանում ես խմբագրել(edit)։ Ավելացրու դատարկ `<div>` թեգ\` նշելու համար այն տեղը, որտեղ ցանկանում եք ցույց տալ ինչ-որ բան React-ով։ Օրինակ.
 
 ```html{3}
-<!-- ... existing HTML ... -->
+<!-- ... գոյություն ունեցող HTML ... -->
 
 <div id="like_button_container"></div>
 
-<!-- ... existing HTML ... -->
+<!-- ... գոյություն ունեցող HTML ... -->
 ```
 
-We gave this `<div>` a unique `id` HTML attribute. This will allow us to find it from the JavaScript code later and display a React component inside of it.
+Մենք այս `<div>`-ին տվեցինք եզակի `id` HTML ատրիբուտ։ Սա մեզ թույլ կտա գտնել այն հետո JavaScript կոդից և պատկերել React կոմպոնենտը նրա ներսում։
 
->Tip
+>Խորհուրդ
 >
->You can place a "container" `<div>` like this **anywhere** inside the `<body>` tag. You may have as many independent DOM containers on one page as you need. They are usually empty -- React will replace any existing content inside DOM containers.
+>Դուք կարող եք տեղադրել սրա նման «կոնտեյներ» `<div>`-եր `<body>` թեգի ներսում **ցանակցած** տեղ։ Մեկ էջում, դուք կարող եք ունենալ միմյանցից անկախ այնքան DOM կոնտեյներներ, որքան կցանկանաք։ Սովորաբար դրանք լինում են դատարկ. React-ը փոխարինելու է DOM կոնտեյներների ներսում ցանկացած գոյություն ունեցող բովանդակություն։
```

Comment:
"ցանակցած" ֊> "ցանԿԱցած"

Reply:
"like this"֊ը վերաբերում ա կոնտոյների՞ն, այսինքն քո գրածի պես "սրա նաման կոնտեյներ", թե՞ տեղադրմանը, այսինքն "տեղադրել այսպես"

Reply:
ես սենց կգրեի "Մեկ էջում, դուք կարող եք ունենալ" ֊> "Դուք կարող եք մեկ էջում ունենալ"

Reply:
"ցանկացած գոյություն ունեցող" ֊> "գոյություն ունեցող ցանկացած"